common/script/script-api/src/main/java/org/thingsboard/script/api/tbel/TbDate.java

```diff
@@ -73,7 +73,9 @@ public String toTimeString() {
     }
 
     public String toISOString() {
-        return isoDateFormat.format(this);
+        synchronized (TbDate.class) {
```

Comment:
Maybe let's use ThreadLocal variable? It should be faster and less resource consuming?